template/src/ontology/Makefile.jinja2

```diff
@@ -393,233 +425,231 @@ no-mirror-refresh-%:
 # ----------------------------------------
 # Some ontologies contain external and internal components. A component is included in the ontology in its entirety.
 
-$(COMPONENTSDIR)/%:
+$(COMPONENTSDIR)/%: | $(COMPONENTSDIR)
 	touch $@
 .PRECIOUS: $(COMPONENTSDIR)/%
 
-
 {% for component in project.components.products %}
 {% if component.source is not none %}
-$(COMPONENTSDIR)/{{ component.filename }}: .FORCE
+.PHONY: component-download-{{ component.filename }}
+component-download-{{ component.filename }}: | $(TMPDIR)
 	if [ $(IMP) = true ]; then $(ROBOT) merge -I {{ component.source }} \
-	annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) -o $@; fi
+	annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) -o $(TMPDIR)/$@.owl; fi
+
+$(COMPONENTSDIR)/{{ component.filename }}:  component-download-{{ component.filename }}
+	cmp -s $(TMPDIR)/component-download-{{ component.filename }}.owl $@; RETVAL=$$?; if [ $$RETVAL -eq 0 ]; then echo "Component identical, ignoring."; else echo "Component is different, updating." && cp $(TMPDIR)/component-download-{{ component.filename }}.owl $@; fi
```

Comment:
From @Clare72 (thanks!)

Missing conditional, see https://github.com/FlyBase/drosophila-anatomy-developmental-ontology/pull/1388#discussion_r806829649

Reply:
Beyond the missing test on the existence of the file, I am wondering about the use of that intermediate variable `RETVAL`. What does it bring compared to a simpler form:

```sh
if cmp -s $(TMPDIR)/component-download-{{ component.filename }}.owl $@ ; then
    echo "Component identical, ignoring."
else
    echo "Component different, updating..."
fi
```

?